processor/src/main/resources/application.yaml

```diff
@@ -15,6 +15,7 @@ kafka:
   consumer:
     group-id: songsapp
   topic-upload: resource-upload
+  topic-proceeded: resource-proceeded
```

Comment:
What do you mean by `proceeded` here?

Reply:
It's all because of my bad English, I meant processed